internal/pkg/checkin/bulk.go

```diff
@@ -167,7 +176,8 @@ func (bc *Bulk) flush(ctx context.Context) error {
 		return nil
 	}
 
-	updates := make([]bulk.MultiOp, 0, len(pending))
+	// make cap pending*2 in case all checked in agents have audit/unenroll attributes that must be removed.
+	updates := make([]bulk.MultiOp, 0, len(pending)*2)
```

Comment:
What is the odds that all of them have it? Could making this larger initial be taking more memory then really will be needed? I know in worst case it will take that much memory, but in a normal case I don't think this should be raised.